packages/react-core/src/modules/core/utils.ts

```diff
@@ -1,13 +1,13 @@
 import { FeedClientOptions } from "@knocklabs/client";
-import { parseISO, formatDistance, Locale } from "date-fns";
+import { parseISO, intlFormatDistance } from "date-fns";
 import { ReactNode } from "react";
 
 export function formatBadgeCount(count: number): string | number {
   return count > 9 ? "9+" : count;
 }
 
 type FormatTimestampOptions = {
-  locale?: Locale;
+  locale?: string | string[];
```

Comment:
Does it make sense to strongly type this to available Locales? Not sure what that involves but feels like that would be a good DX?

Reply:
Agreed it'd be a nice DX improvement but I didn't find a good canonical list e.g. a few of the packages I saw had a lot of helper methods which would bloat the bundle size again. Unfortunately the Intl module doesn't have an out of the box method for this (they also type locale as `string | string[]`. From what I understand, it can depend on the browser implementation as well. Given that, I think we should punt on that.